src/python/pants/backend/python/dependency_inference/parse_python_imports_test.py

```diff
@@ -128,23 +147,76 @@ def test_normal_imports(rule_runner: RuleRunner) -> None:
         rule_runner,
         content,
         expected={
-            "__future__.print_function": ImpInfo(lineno=1, weak=False),
-            "os": ImpInfo(lineno=3, weak=False),
-            "os.path": ImpInfo(lineno=5, weak=False),
-            "typing.TYPE_CHECKING": ImpInfo(lineno=6, weak=False),
-            "requests": ImpInfo(lineno=8, weak=False),
-            "demo": ImpInfo(lineno=10, weak=False),
-            "project.demo.Demo": ImpInfo(lineno=11, weak=False),
-            "project.demo.OriginalName": ImpInfo(lineno=12, weak=False),
-            "multiline_import1.not_ignored1": ImpInfo(lineno=16, weak=False),
-            "multiline_import1.not_ignored2": ImpInfo(lineno=23, weak=False),
-            "multiline_import2.not_ignored": ImpInfo(lineno=26, weak=False),
-            "project.circular_dep.CircularDep": ImpInfo(lineno=29, weak=False),
-            "subprocess": ImpInfo(lineno=32, weak=False),
-            "subprocess23": ImpInfo(lineno=34, weak=False),
-            "pkg_resources": ImpInfo(lineno=36, weak=False),
-            "not_ignored_but_looks_like_it_could_be": ImpInfo(lineno=39, weak=False),
-            "also_not_ignored_but_looks_like_it_could_be": ImpInfo(lineno=45, weak=False),
+            "pkg_resources": ImpInfo(lineno=1, weak=False),
+            "not_ignored_but_looks_like_it_could_be": ImpInfo(lineno=4, weak=False),
+            "also_not_ignored_but_looks_like_it_could_be": ImpInfo(lineno=10, weak=False),
+        },
+    )
+
+
+def test_try_except(rule_runner: RuleRunner) -> None:
+    content = dedent(
+        """\
```

Comment:
All these tests have the import on a single line with the `try` or `except`. I know this is AST code so we should be impervious to line numbers -- but, still, how about changing some of these to take place over multiple lines to have confidence this will never break?

Reply:
(FWIW I tried to smush the lines because otherwise this test case was eating lines like it was at the Line Buffet)

Reply:
(FWIW I tried to smush the lines because otherwise this test case was eating lines like it was at the Line Buffet)

Reply:
Can you give me a snippet of what you're thinking?